.babelrc

```diff
@@ -1,11 +1,20 @@
 {
 	"presets": [
-		"es2015",
+		[ "env", {
+			"targets": {
+				"browsers": [
+					"last 2 versions",
+					"Safari >= 9",
+					"iOS >= 9",
+					"not ie <= 10",
```

Comment:
Would it be simpler to state `ie 11` ? One hopes new IE's aren't to be released in the future 😉